cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -338,6 +339,11 @@ macro (google_cloud_cpp_enable_deps)
     if (asset IN_LIST GOOGLE_CLOUD_CPP_ENABLE)
         list(INSERT GOOGLE_CLOUD_CPP_ENABLE 0 accesscontextmanager osconfig)
     endif ()
+    if (bigtable IN_LIST GOOGLE_CLOUD_CPP_ENABLE)
```

Comment:
Does this enable metrics collection based on the opentelemetry-cpp library being found by CMake?